database/src/migrations/release.0.7/populate_climate_change_initiatives.sql

```diff
@@ -1,4 +1,4 @@
 -- populate_climate_change_initiatives.sql
 insert into climate_change_initiative (name, record_effective_date) VALUES ('Adaptation', now());
 insert into climate_change_initiative (name, record_effective_date) VALUES ('Monitoring', now());
-insert into climate_change_initiative (name, record_effective_date) VALUES ('Mitigation', now());
+insert into climate_change_initiative (name, record_effective_date) VALUES ('Mitigation', now());
```

Comment:
```suggestion
insert into climate_change_initiative (name, record_effective_date) VALUES ('Mitigation', now());

```

Reply:
same as above

Reply:
hmm, it doesnt show it as outdated... so not sure if it is